src/Tizen.NUI/src/public/Common/NUIConstants.cs

```diff
@@ -758,6 +758,29 @@ public enum WindowType
         Dialog
     }
 
+    /// <summary>
+    /// Enumeration of rendering mode for native GL application<br />
+    /// It is used with GLWindow.
+    /// This Enumeration is used to choose the rendering mode.
+    /// It has two options.
+    /// One of them is continuous mode. It is rendered continuously.
+    /// The other is on demand mode. It is rendered by calling RenderOnce().
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public enum NativeGLRendeingMode
```

Comment:
isn't it `NativeGLRenderingMode` ^^;

Reply:
Ah It is my mistake. I will fix it.

Reply:
How about deleting 'Native'?
```csharp
GLRenderingMode
```

Reply:
It makes a sense. I want it is used both GLWindow and GLView.